.github/workflows/test_build.yml

```diff
@@ -40,18 +38,20 @@ jobs:
       matrix:
         os: [macos-latest, ubuntu-latest]
         build-type: [cmake, autotools]
-        compiler: [gcc9, default]  # add clang
+        compiler: [clang10, gcc9, default]
```

Comment:
What's the point of testing clang10, when we're already up to clang11 on the Mac?

Reply:
clang11 doesn't exist yet, as far as I know. The 11 in AppleClang is the version of Xcode.